[2014-08-17 04:03:59] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Method Illuminate\View\View::__toString() must not throw an exception' in D:\All Program\xampp\htdocs\Forex\app\storage\views\3365cd8f57f328f8d2ab566ea3a24f56:0
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2014-08-17 04:05:06] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Method Illuminate\View\View::__toString() must not throw an exception' in D:\All Program\xampp\htdocs\Forex\app\storage\views\3365cd8f57f328f8d2ab566ea3a24f56:0
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2014-08-17 04:06:14] log.ERROR: exception 'InvalidArgumentException' with message 'View [layouts.khachhang] not found.' in D:\All Program\xampp\htdocs\Forex\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:135
Stack trace:
#0 D:\All Program\xampp\htdocs\Forex\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(76): Illuminate\View\FileViewFinder->findInPaths('layouts.khachha...', Array)
#1 D:\All Program\xampp\htdocs\Forex\vendor\laravel\framework\src\Illuminate\View\Environment.php(115): Illuminate\View\FileViewFinder->find('layouts.khachha...')
#2 D:\All Program\xampp\htdocs\Forex\app\storage\views\04b63c8e5482dbedaef96c3f4030a9b5(193): Illuminate\View\Environment->make('layouts.khachha...', Array)
#3 D:\All Program\xampp\htdocs\Forex\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\All Program\...')
#4 D:\All Program\xampp\htdocs\Forex\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\All Program\...', Array)
#5 D:\All Program\xampp\htdocs\Forex\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\All Program\...', Array)
#6 D:\All Program\xampp\htdocs\Forex\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#7 D:\All Program\xampp\htdocs\Forex\vendor\laravel\framework\src\Illuminate\Http\Response.php(70): Illuminate\View\View->render()
#8 D:\All Program\xampp\htdocs\Forex\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#9 D:\All Program\xampp\htdocs\Forex\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#10 D:\All Program\xampp\htdocs\Forex\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#11 D:\All Program\xampp\htdocs\Forex\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View))
#12 D:\All Program\xampp\htdocs\Forex\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array)
#13 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#14 D:\All Program\xampp\htdocs\Forex\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#15 D:\All Program\xampp\htdocs\Forex\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#16 D:\All Program\xampp\htdocs\Forex\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#17 D:\All Program\xampp\htdocs\Forex\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#18 D:\All Program\xampp\htdocs\Forex\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#19 D:\All Program\xampp\htdocs\Forex\index.php(49): Illuminate\Foundation\Application->run()
#20 {main} [] []
[2014-08-17 04:06:28] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Method Illuminate\View\View::__toString() must not throw an exception' in D:\All Program\xampp\htdocs\Forex\app\storage\views\caafab59042e3270de600d0d5854a27c:0
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2014-08-17 04:06:42] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Method Illuminate\View\View::__toString() must not throw an exception' in D:\All Program\xampp\htdocs\Forex\app\storage\views\caafab59042e3270de600d0d5854a27c:0
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2014-08-17 04:07:58] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Method Illuminate\View\View::__toString() must not throw an exception' in D:\All Program\xampp\htdocs\Forex\app\storage\views\e49957e35a876da6d3ff4b8d46ce7076:0
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2014-08-17 04:08:16] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Method Illuminate\View\View::__toString() must not throw an exception' in D:\All Program\xampp\htdocs\Forex\app\storage\views\3365cd8f57f328f8d2ab566ea3a24f56:0
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2014-08-17 04:09:13] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Method Illuminate\View\View::__toString() must not throw an exception' in D:\All Program\xampp\htdocs\Forex\app\storage\views\3365cd8f57f328f8d2ab566ea3a24f56:0
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2014-08-17 04:11:39] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Method Illuminate\View\View::__toString() must not throw an exception' in D:\All Program\xampp\htdocs\Forex\app\storage\views\3365cd8f57f328f8d2ab566ea3a24f56:0
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
